Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Build the manager binary
+FROM golang:1.12.5 as builder
```

Comment:
Should this be bumped to 1.12.6?

Reply:
yeah, i'll do that on a regen, i haven't read through this generated stuff yet